examples/src/main/java/io/dapr/examples/actors/http/DemoActorService.java

```diff
@@ -36,7 +36,8 @@ public static void main(String[] args) throws Exception {
     int port = Integer.parseInt(cmd.getOptionValue("port"));
 
     // Register the Actor class.
-    ActorRuntime.getInstance().registerActor(DemoActorImpl.class, new DefaultObjectSerializer());
+    ActorRuntime.getInstance().registerActor(
+      DemoActorImpl.class, new DefaultObjectSerializer(), new DefaultObjectSerializer());

```

Comment:
Since the examples are using the default, many uses will simply copy and past what's there.  There should be an obvious comment both in the examples *and* in docs that discusses the tradeoffs.

Reply:
I agree. Please, look at the future PR for docs as well.